fury/tests/test_actors.py

```diff
@@ -818,14 +818,15 @@ def test_labels(interactive=False):
     npt.assert_equal(scene.GetActors().GetNumberOfItems(), 1)
 
 
-def test_spheres(interactive=False):
+@pytest.mark.parametrize('interactive, prim', [(False, True), (False, False)])
```

Comment:
We prefer to avoid using `pytest.mark.parametrize` so you can remove it and use a more traditional way